docker-compose.yml

```diff
@@ -180,8 +180,25 @@ services:
     restart: always
     depends_on:
       - opencti
+# Deploy with local persistence for the following services
 volumes:
   esdata:
+    driver: local
+    driver_opts:
+      o: bind
+      type: none
   s3data:
+    driver: local
+    driver_opts:
+      o: bind
+      type: none
   redisdata:
+    driver: local
+    driver_opts:
+      o: bind
+      type: none
   amqpdata:
+    driver: local
+    driver_opts:
+      o: bind
+      type: none
```

Comment:
Default is now to persist data to local disk.

Reply:
I think the data is persistent using the existing volume definition. It is only persistent using the docker volumes instead of being persistent to the local disk
https://docs.docker.com/storage/#choose-the-right-type-of-mount

The only non-persistent approach would be using `tmpfs` (If my docker knowledge doesn't wrong me)

I think we could actually remove the `Data Persistence` section, since the default docker config does provide data persistence. What are your thoughts?

Reply:
Sorry for the delay in response, was away this weekend and just got back. I agree with you; the data persistence with volume mapping in the service is the way to go. I was just persisting what I saw elsewhere on the Notion docs, but I also now see that the bind mounts are a much older way of doing things.

I think by removing this, and re-adding the section:
```
volumes:
  esdata:
  s3data:
  redisdata:
  amqpdata:
```
back in, that would work best.